go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -399,6 +419,20 @@ func (c *Controller) RequestFullSync() {
 		klog.Errorf("Cached lister failed!? %v", err)
 		return
 	}
+	// Resync node IP templates.
+	if c.useTemplates {
+		nodeIPTemplates := []libovsdbops.TemplateMap{
+			{
+				c.nodeTracker.nodeIPV4Template.Name: c.nodeTracker.nodeIPV4Template,
+				c.nodeTracker.nodeIPV6Template.Name: c.nodeTracker.nodeIPV6Template,
```

Comment:
hmm generally everywhere we only do ipv6 stuff if ipv6 is supported..

Reply:
Ack.